resources/translations/es_BO.ts

```diff
@@ -275,35 +275,35 @@
     <message>
         <location filename="../qml/pages/MessagingPage.qml" line="245"/>
         <source>Enter message...</source>
-        <translation>Escribir mensaje...</translation>
+        <translation>A b c</translation>
```

Comment:
Is this correct?

Reply:
Is good. I translated like same in Fernschreiber, also I am a member of sailfish_es so I do not have any complains.